OpenOversight/app/utils.py

```diff
@@ -241,6 +243,13 @@ def filter_by_form(form, officer_query, is_browse_filter=False):
                                                     Officer.gender == 'Not Sure',
                                                     Officer.gender == None))  # noqa
 
+    if form['year']:
+        year = int(form['year'])
+        min_employment_date = datetime.date(year, 1, 1)
+        if year in (2019, 2018, 2017, 2016):
```

Comment:
Also here it would probably be better to query the range of possible dates from the database rather than a static list.

Reply:
Agreed!